### DISCUSSION TITLE
AI features that don't depend on the proprietary Kythera AI

### DISCUSSION THREAD

**Discussion by AreebDurrani** [o3de/rfcs#35] [Ideas]
I see that the AI gem featured on the home page is Kythera AI. While it seems like a great tool, unfortunately it is not available under a free and open source license. The LmbrCentral gem already provides navmesh functionality. I propose we retain this feature and expand on it with behavior trees, spatial queries, and so on. Of course, Kythera will still be available to those who want to use it.

**Comment by podulator** [comment]
Yes, it would be wonderful if it supported the OpenAI Gym Interface, or similar : http://gym.openai.com/ and agents capable of using lidar , sound and camera sensors

**Comment by PeteX** [comment]
The situation with the AI is a bit unfortunate I think.  Unreal is free (in a financial sense) for almost all indie projects, because the first $1M in revenue is royalty-free.  With o3de, you are almost certain to want a navmesh, even if you don't need other AI facilities.  That means you owe 3% royalties, after some small deductions.  The result is that you actually pay more for o3de, even though most of it is open source.

I've seen a few people saying the LmbrCentral gem provides a navmesh, but it doesn't seem to be documented for o3de, only for Lumberyard—unless I'm missing something.  Would it be possible to document it, or reactivate it in the code if it no longer works?  If that's not feasible, Godot has a navmesh implementation that is available under a liberal licence.

Thank you for your work on o3de!
